doc-Migration_Analytics_Guide/cfme/MA_1.0/master.adoc

```diff
@@ -27,28 +27,33 @@ Migration Analytics is available as a technology preview for Red Hat CloudForms
 
 include::modules/Migration-analytics-workflow.adoc[leveloffset=+1]
 
-== Migration Analytics requirements
+=== Migration Analytics requirements
 
-include::modules/Supported-browsers.adoc[leveloffset=+2]
-include::modules/Supported-platforms.adoc[leveloffset=+2]
-include::modules/SSA-support.adoc[leveloffset=+2]
+include::modules/Supported-browsers.adoc[leveloffset=+3]
+include::modules/Supported-platforms.adoc[leveloffset=+3]
+include::modules/SSA-support.adoc[leveloffset=+3]
 
-[id='Deploying-migration-analytics_{context}']
-== Deploying Migration Analytics
+== Installing Red Hat CloudForms and the VMware Virtual Disk Development Kit
 
-You can deploy Migration Analytics by installing CloudForms on VMware, adding the VMware provider, and configuring CloudForms.
+You can install the Red Hat CloudForms appliance on VMware vSphere. You then download and install the VMware Virtual Disk Development Kit (VDDK) SDK. VDDK is required for SmartState analysis of the VMware virtual machines.
 
 include::modules/Installing-cloudforms.adoc[leveloffset=+2]
+include::modules/Installing-vddk.adoc[leveloffset=+2]
+
+== Configuring Red Hat CloudForms for Migration Analytics
+
+In the CloudForms user interface, you can add VMware vCenter as a provider, assign a control policy to the provider, and configure CloudForms for Migration Analytics.
```

Comment:
"...you can add VMware vCenter as a provider..." feels like it is missing a word before provider. At the risk of being repetitive, perhaps "...as an infrastructure provider..."